app/templates/_gitignore

```diff
@@ -5,6 +5,7 @@ ui/fonts
 ui/styles/*.css
 deploy/local.properties
 local.json
+ecosystem.json
```

Comment:
Not sure this really needs to be in here. There will be no pwds in there, if assuming ssh keys, and not running Roxy post-deploy. There might be a username, but pm2 deploy is only used to deploy to dev or prod, and that is usually done by just one person. If someone else needs to take over, he/she can edit the file temporarily. At least server name and git location are already entered, so it would save colleagues time..